### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by tumainimaganiko**
In this project I did:

- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.
- Use a JavaScript array to store all of the information for all projects.
- For each project, I store pieces of data in a JavaScript object.
- Update the main page so that the projects section is created dynamically using the information stored in that JavaScript object.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -288,6 +290,27 @@ footer ul {
...
 }
 
+.modal > ul li {
+  padding: 12px;
+  margin: 6px;
+  border: 1px solid black;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
```
Comment (Yerimah) [inline styles.css:313]: - [ ] Hey guys, I must commend your effort in implementing some of the requirements for this project. However, there are some issues I think you should fix. It is very important that your popup follows the design template provided e.g., font, colors, images, text, and margins. Consistency is an essential design requirement. 

- Make the buttons on your popup modal look exactly like the buttons on the Figma template. Ensure that the  button text and icon are properly aligned
- Make sure the text on the right properly aligns with the image as it is on the Figma design and Capitalize the Header text on the popup.
- Reduce the space between the language buttons and the Header text.

> Your Design
yourFigtem

> Figma Design
Figtem

File: styles.css [hunk 2]
```diff
@@ -522,6 +516,121 @@ footer ul {
...
+  background-color: black;
+  opacity: 0.8;
+  backdrop-filter: blur(5px);
+  pointer-events: all;
+  display: none;
+}
+
+.modal {
+  border: 1px solid black;
+  z-index: 1000;
```
Comment (Yerimah) [inline styles.css:547]: - [ ] On the mobile version I would also suggest that you try to make your design as close to the Figma design as much as possible. 
- Add a class to the `div` container containing the `p tag` and the buttons. that would make it easy for you to align the container with the image.
- Make sure the button text is aligned with the button icon. Include icons for both buttons

> Your Design
mobilfig

> Figma Design
mobilReal

**Review by Yerimah — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️ 

Hi Team, 👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f4d634a5755566852566e493378656f516f6d2f67697068792e676966


## Highlights
- No linter error ✔️ 
- Descriptive PR title ✔️ 
- Descriptive PR summary ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lookingfor00 — APPROVED** [review]
# Approved 🟢
Hi @tumainimaganiko, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
allfather-approves-approved

## Highlights
- Project look awsome 🍏 🌷 
- Code is clean and well organized 🧼 💯 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@Trast00) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**